catalog/spatial/geocoding/spatial-geocoding-query/src/test/java/org/codice/ddf/spatial/geocoding/query/TestGeoNamesQueryLuceneIndex.java

```diff
@@ -296,23 +294,22 @@ public void testNearestCitiesNullMetacard() {
 
     @Test(expected = IllegalArgumentException.class)
     public void testNearestCitiesNegativeRadius() {
-        directoryIndex.getNearestCities(new MetacardImpl(), -1, 1);
+        directoryIndex.getNearestCities("", -1, 1);
```

Comment:
why are we using different numbers here? (-1, 1) shouldn't the test be identical to how we call it?